utilities/scripts/xmlPagsToEntityPags.groovy

```diff
@@ -0,0 +1,199 @@
+/**
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * This file is a groovy script that parses the PAGSGroupStoreConfig.xml and creates entity-PAGS files that
+ * can be imported into an existing uPortal database, creates a SQL script to update the references to the
+ * XML PAGS groups to entity-PAGS groups, and provides some instructions to the user on updating
+ * compositeGroupService.xml to switch from XML PAGS to entity PAGS.
+ *
+ * This script is best IMHO executed directly by groovy as:
+ * groovy xmlPagsToEntityPags.groovy pathToPAGSGroupStoreConfig.xml directoryToWriteEntityFilesTo
+ *
+ * However it can also be invoked via ant as:
+ * ant -Dmaven.test.skip=true -Ddir=directoryToWriteEntityFilesTo data-import
+ * in which case to this groovy script it will look as if invoked as
+ * groovy -s pathToscript xmlPagsToEntityPags.groovy pathToPAGSGroupStoreConfig.xml directoryToWriteEntityFilesTo
+ *
+ * @since 4.2.0
+ */
+
+import groovy.xml.MarkupBuilder
+import org.xml.sax.EntityResolver
+import org.xml.sax.InputSource
+import org.xml.sax.SAXException
+
+// Extra blank output in case script is invoked through ant to segregate it's output better
+println()
+println()
```

Comment:
I would probably do this with one line -- like maybe `print('\n\n');` -- but this approach is fine.